mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupRegistration.java

```diff
@@ -112,6 +117,12 @@ static String groupName(Group group) {
     }
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GroupRegistration.class);
+    private static final String RETRY_COUNT = "retry-count";
+    private static final int DEFAULT_RETRY_COUNT = 0;
+
+    private static final double DEFAULT_JITTER_FACTOR = 0.5;
```

Comment:
Should you POJO for these properties

Reply:
Yep, a nice configuration object would be awesome!